webbpsf/webbpsf_core.py

```diff
@@ -1232,7 +1232,7 @@ def _apply_jitter(self,  result, local_options=None):
 
         This adds options to model JWST coarse point ("PCS=Coarse") under
         two sets of assumptions:
-            "PCS=Coarse": 67 mas Gaussian jitter, as advised by Nelan & Maghamni based on
+            "PCS=Coarse": 67 mas Gaussian jitter, as advised by Nelan & Maghami based on
```

Comment:
This seems to match another instance of what I think is the same name in this repo.